composer.json

```
@@ -19,8 +19,8 @@
         "classmap": [ "app/AppKernel.php", "app/AppCache.php" ]
     },
     "require": {
-        "php": ">=5.6.19",
-        "symfony/symfony": "^3.3.0",
+        "php": "7.1.*",
```

```suggestion
        "php": ">7.1",
```

no need to exclude higher (not deprecated) PHP versions